openwisp_controller/geo/api/views.py

```diff
@@ -43,7 +43,8 @@ def get_object(self, *args, **kwargs):
     def create_location(self, device):
         location = Location(name=device.name,
                             type='outdoor',
-                            organization=device.organization)
+                            organization=device.organization,
+                            is_mobile=True)
```

Comment:
Why is `is_mobile=True` added here? :confused:

Reply:
https://github.com/openwisp/django-loci/commit/2b2b5d3cbd9a97e369c3577056dc9f4aec389f25

Geometry cannot be left as None if is_mobile is not true.

Reply:
this was most probably unintended and caused by a lack of strictness in django-loci.
I think it's safer to add geographic coordinates rather than set `is_mobile=True`, which is kind of a special case instead of the norm.

Can you please add any geographic coordinates to `geometry`? You can use a Point() with any coordinates.

Reply:
I think it would be better to change the Location model in `openwisp_controller.geo.models`
What do you think would be a better option?

Reply:
Sorry, I reviewed the code more closely and I misinterpreted the change. It's good as is for now. If it needs further improvement we'll find out and eventually do it in the future.